chromadb/test/test_api.py

```diff
@@ -1368,3 +1368,22 @@ def test_multiple_collections(api_fixture, request):
 
     assert results1["ids"][0][0] == ids1[0]
     assert results2["ids"][0][0] == ids2[0]
+
+    
+@pytest.mark.parametrize("api_fixture", test_apis)
+def test_upsert(api_fixture, request):
```

Comment:
Can we add testing coverage for:

- Delete an embedding and then upsert it with the same id. Since HNSW deletes are soft I want to implicitly check that behavior works correctly. 
- The increment_index = false case.